src/Type/WPEnumType.php

```diff
@@ -64,9 +65,9 @@ private static function prepare_values( $values, $type_name ) {
 			if ( ! empty( $value['description'] ) && is_callable( $value['description'] ) ) {
 				$description = $value['description']();
 			} else {
-				$description = is_string( $value['description'] ) ? $value['description'] : '';
+				$description = isset( $value['description'] ) && is_string( $value['description'] ) ? $value['description'] : '';
 			}
-			$values[ $key ]['description'] = $description;
+			$values[ $key ]['description'] = $description ?? '';
```

Comment:
Do we still need this foreach loop if we're preparing_for_introspection later in this method? (or vice versa)

Reply:
@justlevine this is to prepare the values of the enum, the other is for the Enum Type description